verifier/fake/fakeverifier.go

```diff
@@ -182,3 +196,41 @@ func extractPCRBank(attestation *attest.Attestation, hashAlgo tpm.HashAlgo) (*re
 	}
 	return nil, fmt.Errorf("no PCRs found matching hash %s", hashAlgo.String())
 }
+
+func fakePrivateKey() crypto.Signer {
+	// Hardcoded fake key
+	const privateKeyPEM = `-----BEGIN PRIVATE KEY-----
```

Comment:
this should go in a file, perhaps under something like `verifier/fake/signer_rsa`

Reply:
One thing you could try is embedding and exporting in something like `verifier/fake/data.go`. Then add the verifier module to your go.mod in the happypath test directory